integration-tests/interactions/bot/daily_ci_enroll/registration_complete.jinja

```diff
@@ -0,0 +1,38 @@
+{% import 'bot/macros.jinja' as bot %}
+
+[
+    {
+        "text": "Thanks. I have recorded all your answers in your personal profile."
+    },
+    {
+        "text": "Your registration to the daily check-in is now completed."
+    },
+    {
+        "text": "I will check-in with you everyday around this time to see how you're doing, monitor your symptoms and direct you to the right resources."
+    },
```

Comment:
Is there a way to separate these messages and concatenate them after? These first 3 are always sent after enrollment, but the messages after depend on where we were before enrollment (suspect, returning or tested positive flow). There are many cases like this related to self-isolation and home-assistance sub-flows. If we have to copy them for each flow and something changes we'll have to change them in many places. It wouldn't be dramatic though, only sub-optimal.

Reply:
Yes, it bothered me too. It should be possible to extract them using Jinja macros. I'll figure something cute in another PR.